libs/mstable/flatcoin/src/components/TradingPanel/index.tsx

```diff
@@ -26,27 +27,26 @@ const TRADING_TAB_INDEX_MAP = TABS.reduce<Record<number, PositionType>>(
 );
 
 const useTradingPanel = () => {
-  const intl = useIntl();
   const [type, updateFlatcoinType] = useFlatcoinType();
   const [, updateTradingType] = useTradingType();
+  const { tokens } = useFlatcoin();
 
   const onTabChange = (index: number) => {
     const type = TRADING_TAB_INDEX_MAP[index];
     updateFlatcoinType(type);
-    type === 'leveragedeth' && updateTradingType('deposit');
+    type === 'leveraged' && updateTradingType('deposit');
   };
-  const tabNameMap: Record<PositionType, string> = useMemo(
-    () => ({
-      flatcoin: intl.formatMessage({
-        defaultMessage: 'Flatcoin',
-        id: 'Ew5cbe',
-      }),
-      leveragedeth: intl.formatMessage({
-        defaultMessage: 'Leveraged ETH',
-        id: 'xSqIpD',
-      }),
-    }),
-    [intl],
+
+  const tabNameMap = useMemo(
+    () =>
+      TABS.reduce(
+        (acc, type) => ({
+          ...acc,
+          [type]: tokens[POSITION_TYPE_TOKEN_KEY_MAP[type]].name,
```

Comment:
It will be better to show `Leveraged ${collateral.symbol}` as a tab name for leveraged trading